google-datacatalog-rdbms-connector/README.md

```diff
@@ -189,6 +190,61 @@ and `_get_path_to_num_rows_query`.
 You can see working examples of implementing optional queries in the connectors code for PostgreSQL and MySQL. 
 Please also refer to these samples to see how user configuration file `ingest_cfg.yaml` should look like.
 
+### 4.2 Add support for SQL Objects
+You can use the SQL Objects mechanism to ingest SQL objects like Functions, Stored Procedures,
+Views, Materialized Views... from the SQL connector. This mechanism uses naming convention to locate
+a SQL query and a metadata definition file.
+
+You can see working examples of implementing SQL Objects in the test sources. 
+To enable it you need 3 configurations:
+1. [ingest_cfg.yaml](tests/google/datacatalog_connectors/rdbms/test_data/sql_objects_ingest_cfg.yaml)
```

Comment:
The link is displayed as **ingest_cfg.yaml** but points to `tests/google/datacatalog_connectors/rdbms/test_data/sql_objects_ingest_cfg.yaml`. And there is a `ingest_cfg.yaml` file at `tests/google/datacatalog_connectors/rdbms/test_data`. It is a bit confusing... What are the relationships and expected content for these files?

Reply:
On the next PR I will update this to point to a sample implementation on the SAP HANA connector, while there are no connectors using this, I pointed out the test_data.